Dockerfile

```diff
@@ -1,54 +1,48 @@
-FROM google/debian:wheezy
-MAINTAINER u@gogs.io
-
-RUN echo "deb http://ftp.debian.org/debian/ wheezy-backports main" >> /etc/apt/sources.list && \
-	apt-get update -qqy && \
-	apt-get install --no-install-recommends -qqy \
-	curl build-essential ca-certificates git \ 
-	openssh-server libpam-dev && \
-	apt-get autoclean && \
-    apt-get autoremove && \
-    rm -rf /var/lib/apt/lists/*
-
-ENV GOROOT /goroot
-ENV GOPATH /gopath
-ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
-
-COPY . /gopath/src/github.com/gogits/gogs/
-WORKDIR /gopath/src/github.com/gogits/gogs/
+FROM debian:jessie
```

Comment:
Ah! If you say the FROM line:
`FROM debian:jessie`

This base distro is officially supported by docker team, and is most recent than `google/debian:wheezy`.

It fixes the issues reated with SSH and ED25519 keys. :smile: